File: Core/Entities/Review.cs
namespace Core.Entities
{
    public partial class Review
    {
        public int Id { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; }
        public DateTime CreatedDateTime { get; set; } = DateTime.UtcNow;

        public int ClientId { get; set; }
        public virtual AppUser Client { get; set; }

        public int BarberId { get; set; }
        public virtual AppUser Barber { get; set; }     
    }
}

File: UI/barbershop_mobile/lib/models/review.dart
import 'package:json_annotation/json_annotation.dart';

part 'review.g.dart';

@JsonSerializable()
class Review {
  int? id;
  int? rating;
  String? comment;
  int? clientId;
  String? clientFirstName;
  String? clientLastName;
  DateTime? createdDateTime;

  Review({
    this.id,
    this.rating,
    this.comment,
    this.clientId,
    this.createdDateTime,
  });

  factory Review.fromJson(Map<String, dynamic> json) => _$ReviewFromJson(json);

  Map<String, dynamic> toJson() => _$ReviewToJson(this);
}
